invenio_app_ils/ui/src/invenio_app_ils/forms/components/BaseForm/BaseForm.js

```diff
@@ -55,7 +61,7 @@ export class BaseForm extends Component {
 
   render() {
     return (
-      <>
+      <Container>
         <Grid>
           <Grid.Row centered>
```

Comment:
Not sure you need a grid here to center the title

Reply:
Yes that was the easiest way to do this...But for sure there should another way I just chose that to move forward. On top of that I am not sure after you iterate through that this title is gonna still be at the center :P